core/frontend/src/components/bridges/BridgetUpdater.vue

```diff
@@ -20,12 +20,21 @@ import { callPeriodically } from '@/utils/helper_functions'
 
 export default Vue.extend({
   name: 'BridgetUpdater',
+  data() {
+    return {
+      prefetched_bridges: false,
+      prefeteched_serial_ports: false,
```

Comment:
This two variables looks more as a state control (that should be an enum) than two non relatable booleans